src/yaml/mapping.cr

```diff
@@ -29,16 +29,15 @@ module YAML
   #
   # ```
   # employee = Employee.from_yaml("title: Manager\nname: John\nage: 30")
-  # employee.age # => undefined method 'age'.
+  # employee.age # undefined method 'age'. (compile error)
   #
-  # employee = Employee.from_yaml("title: Manager")
-  # # => ParseException: missing yaml attribute: name
+  # Employee.from_yaml("title: Manager") # raises YAML::ParseException
   # ```
   #
   # You can also define attributes for each property.
   #
   # ```
-  # class Employee
+  # class Employer
```

Comment:
Well, I left `Employer` as is because it realized that it would be better to use same names in examples of `YAML.mapping` and `JSON.mapping`.
And the refactor should be a new other PR.